distributed/src/test/scala/asobu/distributed/util/TestRerequests.scala

```diff
@@ -0,0 +1,43 @@
+package asobu.distributed.util
+
+import javax.inject.{Inject, Provider, Singleton}
+
+import akka.actor._
+import akka.cluster.Cluster
+import asobu.distributed.gateway.{Gateway, Endpoint}
+import play.api.libs.json.Json
+import play.api.{Configuration, Environment}
+import play.api.inject.Module
+import play.api.mvc._
+
+import scala.concurrent.{Await, Promise}
+import scala.concurrent.duration._
+
+case class MockRequest(path: String, headers: Headers = Headers()) extends Request[AnyContent] {
+  def body: AnyContent = AnyContentAsJson(Json.obj())
+  def secure: Boolean = ???
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Usage of ??? operator](https://www.codacy.com/app/kailuo-wang/asobu/file/2251247478/issues/source?bid=3037259&fileBranchId=3178877#l18)